icons/balloon.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M12 16V16.6C12 18 13 19 14.4 19H15.6C17 19 18 20 18 21.4V22" />
+  <path d="M12 6C13 6 14 7 14 8" />
+  <path d="M15.8492 13.5763C14.5587 15.2049 13.0678 16 12 16C10.9292 16 9.4413 15.2049 8.15081 13.5802C6.89474 11.9989 6 9.83299 6 7.51655C6 4.52014 8.63259 2 12 2C15.3674 2 18 4.52014 18 7.51655C17.9806 9.71024 17.2255 11.8379 15.8492 13.5763" />
```

Comment:
Hi @peteruithoven,

Cute icon, but I think this could do with better optimisation and the use of arcs instead of cubic curves:
```suggestion
  <path d="M12 16v.6c0 1.4 1 2.4 2.4 2.4h1.2c1.4 0 2.4 1 2.4 2.4v.6" />
  <path d="M12 6a2 2 0 0 1 2 2" />
  <path d="M18 8c0 4-3.5 8-6 8s-6-4-6-8a6 6 0 0 1 12 0" />
```

Reply:
Thanks for all the optimizations!  
I don't think it matters much but out of curiosity why also move the bottom of the rope 1 pixel to the left?

Reply:
> Thanks for all the optimizations! I don't think it matters much but out of curiosity why also move the bottom of the rope 1 pixel to the left?

You can move it 1px to the right, I mostly care about using arcs instead of curves whenever possible.